### PR TITLE
Support for DSC06106-ZWEU

### PR DESCRIPTION
**Pull request by rebtor**
Add support for DSC06106-ZWEU

### CODE REVIEW

**Comment by caseda** [comment]
you have copied over the same exact id's from the ZW075 driver implementation? Product ID's 24 and 75

**Comment by rebtor** [comment]
You are absolutely right. I screwed up the app.json while creating the fork. Fixed it.
thanks.

**Comment by jeroenvollenbrock** [comment]
Hi! Thank you for submitting this PR. Unfortunately, we don't have the device available for testing. Can you perhaps get a few users to test your branch and report their feedback here?

File: app.json [hunk 1]
```diff
@@ -1508,6 +1508,387 @@
...
+          "instruction": {
+            "en": "Press the (action) button on the Smart Switch to enter learn mode",
+            "nl": "Druk op de (actie) knop op de Smart Switch om de leer modus te activeren"
+          }
+        },
+        "associationGroups": [
+          1,
+          2
+        ],
+        "defaultConfiguration": [
```
Comment (RobinBol) [inline app.json]: Are all these default configurations really necessary? As in, are they needed for the device to work out-of-the-box and do they differ from the default setting of the device?

File: drivers/DSC06106/driver.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+				},
+			},
+		],
+		measure_power: {
+			command_class: 'COMMAND_CLASS_METER',
+			command_get: 'METER_GET',
+			command_get_parser: () => ({
+				'Sensor Type': 'Electric meter',
+				'Properties1': {
+					'Scale': 0,
```
Comment (RobinBol) [inline drivers/DSC06106/driver.js]: Measure power should use `Scale: 2` to receive Wattage reports. The `Sensor Type` can be removed by the way.

File: drivers/DSC06106/driver.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+				return null;
+			},
+		},
+		meter_power: {
+			command_class: 'COMMAND_CLASS_METER',
+			command_get: 'METER_GET',
+			command_get_parser: () => ({
+				'Sensor Type': 'Electric meter',
+				'Properties1': {
+					'Scale': 2,
```
Comment (RobinBol) [inline drivers/DSC06106/driver.js]: Meter power should use `Scale: 0` to receive kWh reports. The `Sensor Type` can be removed here as well.

**Comment by jeroenvollenbrock** [comment]
@rebtor Could you please incorporate the feedback @RobinBol gave you into a new commit so we can proceed with this PR?

**Pull request closed without merging**